sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample12_GetSettingsUsingTags.md

```diff
@@ -0,0 +1,54 @@
+# Get Settings With Tags
+
+This sample demonstrates how to fetch configuration settings from the Azure App Configuration service using tags. To get started, you'll need a connection string to Azure App Configuration. See the [README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/appconfiguration/Azure.Data.AppConfiguration/README.md) for links and instructions.
+
+ ## Create a ConfigurationClient
+
+To interact with Azure App Configuration, you need to instantiate a `ConfigurationClient`. You can use either an endpoint URL and a [`TokenCredential`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/README.md#credentials), or a connection string.
+
+For the sample below, you can set `connectionString` in an environment variable, a configuration setting, or any way that works for your application. The connection string is available from the App Configuration Access Keys view in the Azure Portal.
+
+```C# Snippet:AzConfigSample12_CreateConfigurationClient
+var client = new ConfigurationClient(connectionString);
+```
+
+## Create configuration setting
+
+First, create an instance of `ConfigurationSetting` with a key, value, label, and tags.
+
+```C# Snippet:AzConfigSample12_CreateConfigurationSettingAsync
+ var betaEndpoint = new ConfigurationSetting("endpoint", "https://beta.endpoint.com", "beta")
+ {
+     Tags = { { "someKey", "someValue" } }
+ };
+```
+
+There are two ways to create a Configuration Setting
```

Comment:
The purpose of this sample is to show you how to get settings by tag.   I don't think we want to go into a deeper discussion about how to create.   Let's just choose one approach.